rest-service/manager_rest/test/base_test.py

```diff
@@ -303,6 +303,10 @@ def setUpClass(cls):
         cls.client = cls.create_client()
 
     def setUp(self):
+        test_context = server.app.test_request_context()
+        test_context.push()  # type: ignore
```

Comment:
What makes us need ti not check type here?

Reply:
hmm:
```
manager_rest/test/base_test.py:307: error: "ContextManager[RequestContext]" has no attribute "push"
manager_rest/test/base_test.py:308: error: "ContextManager[RequestContext]" has no attribute "pop"
``` I'm not sure why. It does have those.

Reply:
Ah, flask 2.x fixes this